src/Helper/Helper_PDF_List_Table.php

```diff
@@ -361,6 +365,6 @@ public function column_name( $item ) {
 	 * @since 4.0
 	 */
 	public function no_items() {
-		printf( esc_html__( "This form doesn't have any PDFs. Let's go %1\$screate one%2\$s.", 'gravity-forms-pdf-extended' ), "<a href='" . add_query_arg( [ 'pid' => 0 ] ) . "'>", '</a>' );
+		printf( esc_html__( "%3\$sThis form doesn't have any PDFs. Let's go %1\$screate one%2\$s.%4\$s", 'gravity-forms-pdf-extended' ), "<a href='" . add_query_arg( [ 'pid' => 0 ] ) . "'>", '</a>', '<label>', '</label>' );
```

Comment:
Don't want to complicate the translation string further with more `printf` placeholders Instead, wrap the entire output with your `<span>` tag.

Reply:
Updated.